### DISCUSSION TITLE
Redes sociais

### DISCUSSION THREAD

**Discussion by christiansousadev** [EasyRoomCompany/Guardian#69] [General]
Preciso do github e linkedin de todos para serem direcionadas na homepage na apresentação da nossa equipe

**Comment by ivannaprata** [comment]
http://linkedin.com/in/ivannaprataanalistadedados

**Comment by mikaelmonteirodev** [comment]
MIKAEL MONTEIRO
https://github.com/mikaelmonteirodev
https://www.linkedin.com/in/mikaelmonteiro/

KEVIN SOUSA
https://github.com/kevinDsousa
https://www.linkedin.com/in/kevindik-programmer/

IVANNA PRATA
https://github.com/ivannaprat
https://www.linkedin.com/in/ivannaprataanalistadedados/

JOEL LACERDA
https://github.com/joellacerda
https://www.linkedin.com/in/joellacerdaol/

CHRISTIAN SOUSA
https://github.com/christiansousadev
https://www.linkedin.com/in/christiansousadev/

ENZO GOMEZ
https://github.com/enzogomezz
https://www.linkedin.com/in/enzogomess/

FABIANO MELO
https://github.com/fabianopmelo
https://www.linkedin.com/in/fabiano-melo-5a752294/

GABRIEL SOUSA
https://github.com/gabrielsousarod
https://www.linkedin.com/in/gabriel-sousa-406889276/
